김보민/RestApiStudy/src/main/java/net/skhu/restapistudy/repository/MemoryMovieRepository.java

```diff
@@ -0,0 +1,40 @@
+package net.skhu.restapistudy.repository;
+
+import net.skhu.restapistudy.domain.Movie;
+import org.springframework.stereotype.Repository;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+//CRUD
+@Repository
+public class MemoryMovieRepository implements MovieRepository{
+    private static Map<Long, Movie> store = new HashMap<>();
+
+    @Override
+    public void save(Movie movie){
+        store.put(movie.getId(), movie);
+    }
+
+    @Override
+    public Movie findById(Long id){
+        return store.get(id);
+    }
+
+    @Override
+    public List<Movie> findAll(){
+        return store.values().stream().sorted(Comparator.comparing(Movie::getCount).reversed()).toList();
+    }
```

Comment:
```suggestion
    @Override
    public List<Movie> findAll(){
        return store.values().stream()
                .sorted(Comparator.comparing(Movie::getCount)
                        .reversed())
                .toList();
    }
```
이렇게 줄바꿈 해주면 더 가독성 좋아질 것 같아요~~